src/mock/data/resources.ts

```diff
@@ -0,0 +1,39 @@
+import { ResourceConfigurationsType } from '../../resource-manager';
```

Comment:
We can have `app-config` use this when running in mock mode, this will allow us to see the dispensers in `npm run start:mock`.

Reply:
Done!